static/help/cs/help.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html><head><meta charset="UTF-8" /><meta name="viewport" content="initial-scale=1.0" /><link rel="stylesheet" href="../help.css" /></head><body><ul id="top">
+<html><head><meta charset="UTF-8" /><meta name="color-scheme" content="dark light"><meta name="viewport" content="initial-scale=1.0" /><link rel="stylesheet" href="../help.css" /></head><body><ul id="top">
```

Comment:
these files are generated by https://github.com/deltachat/deltachat-pages/blob/master/tools/create-local-help.py , so changes should go there, maybe conditional for desktop only